lib/utils/process-jobs.ts

```diff
@@ -206,6 +204,7 @@ module.exports = function(extraJob) {
       debug('[%s:%s] nextRunAt is in the past, run the job immediately', job.attrs.name, job.attrs._id);
       runOrRetry();
     } else {
+      // @ts-expect-error
```

Comment:
Nit; I'd elaborate why this was suppressed here, and in other places where used.